app/utils/log.js

```diff
@@ -10,6 +10,10 @@ export const { leaveBreadcrumb } = bugsnag;
 
 let metadata = {};
 
+export const trackUserEvent = ({ eventName, payload }) => {
```

Comment:
Follow `logEvent`'s signature in order to remove clutter: https://v5.rnfirebase.io/docs/v5.x.x/analytics/reference/analytics#logEvent